client/my-sites/people/people-invites/index.jsx

```diff
@@ -77,13 +77,13 @@ class PeopleInvites extends PureComponent {
 
 		return (
 			<Main className="people-invites">
-				<PageViewTracker path="/people/invites/:site" title="People > Invites" />
+				<PageViewTracker path="/people/invites/:site" title="Users > Invites" />
```

Comment:
Not sure if we should change the tracked title, since that can disrupt folks actively monitoring these (and possibly break any funnel currently using them?). It may be safe to keep tracking the old People title, even if that's not the title a user sees.